evmbin/src/main.rs

```diff
@@ -14,7 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Parity EVM interpreter binary.
+//! Parity EVM Interpreter Binary.
+//!
+//! ## Overview
+//!
+//! The Parity EVM interpreter binary is an additional tool in the Parity
+//! Ethereum toolchain. It is an EVM implementation for Parity Ethereum that
+//! may be used to run a standalone version of the EVM interpreter.
+//!
+//! ## Usage
+//!
+//! Build Parity Ethereum from source, then start it with:
+//! ```bash
+//! cd evmbin && cargo build --release && cd ..
```

Comment:
something like

```
cargo build -p evmbin
```

and is much simpler

Reply:
i've pushed a commit addressing your comment to simplify it

Reply:
sorry, i've pushed a commit fixing it now so it's:
```
//! ```bash
//! cargo build -p evmbin --release
//! ./target/release/parity-evm --help
//! ```
```

Reply:
Is that ok @soc1c ?